src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -133,10 +203,31 @@ function edmfx_sgs_flux_tendency!(
                     edmfx_upwinding,
                 )
             end
+
+            # diffusive flux
+            ᶜdivᵥ_ρq_tot = Operators.DivergenceF2C(
+                top = Operators.SetValue(C3(FT(0))),
+                bottom = Operators.SetValue(
+                    sfc_conditions.ρ_flux_q_tot[colidx],
+                ),
+            )
+            @. Yₜ.c.ρq_tot[colidx] -= ᶜdivᵥ_ρq_tot(
+                -(ᶠρaK_h[colidx] * ᶠgradᵥ(ᶜspecific.q_tot[colidx])),
+            )
         end
+
+        # diffusive flux
+        ᶠρaK_u = p.ᶠtemp_scalar
+        @. ᶠρaK_u[colidx] = ᶠinterp(Y.c.ρ[colidx]) * ᶠinterp(ᶜK_u[colidx])
+        ᶜdivᵥ_uₕ = Operators.DivergenceF2C(
+            top = Operators.SetValue(C3(FT(0)) ⊗ C12(FT(0), FT(0))),
+            bottom = Operators.SetValue(sfc_conditions.ρ_flux_uₕ[colidx]),
+        )
+        @. Yₜ.c.uₕ[colidx] -=
+            ᶜdivᵥ_uₕ(-(ᶠρaK_u[colidx] * ᶠgradᵥ(Y.c.uₕ[colidx]))) / Y.c.ρ[colidx]
```

Comment:
Identical to eqn for `EDMFX` type, and to the `vertical diffusion` tendency in the dycore prognostic equations.